spec/css/moz_document/functions/interpolated/warning

```diff
@@ -2,68 +2,119 @@ DEPRECATION WARNING on line 1, column 1 of /sass/spec/css/moz_document/functions
 @-moz-document is deprecated and support will be removed from Sass in a future
 relase. For details, see http://bit.ly/moz-document.
 
-@-moz-document url(#{"sass-lang.com"}) {
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  ,
+1 | / @-moz-document url(#{"sass-lang.com"}) {
+2 | |   a {type: unquoted full url}
+3 | | }
+4 | | @-moz-document url(#{sa + ss}-lang.com) {
+  | '^
```

Comment:
All the indicators in this file moved locations. Which version is more accurate?